airflow-core/src/airflow/models/dagrun.py

```diff
@@ -435,6 +435,9 @@ def duration(cls, session: Session = NEW_SESSION) -> Case:
 
         return case(when_condition, else_=None)
 
+    # Class-level expression for use in select statements
```

Comment:
Thanks for the change, this looks cleaner ! One smal nit is this is now class level variable defined in the middle of the class. Maybe we can convert this to a property since it is expression, it can be easily calculated under property since it is not populating any data but generating sql expression